debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectFieldInfo.java

```diff
@@ -213,10 +234,15 @@ boolean isSuperclassBackfillSlotAvailable() {
 	 */
 	int calculateFieldDataStart() {
 		int fieldDataStart = getSuperclassFieldsSize();
+		boolean doubleAlignment = (totalDoubleCount > 0);
+		if (ValueTypeHelper.areValueTypesSupported()) {
+			doubleAlignment = doubleAlignment || (totalFlatFieldDoubleBytes > 0);
```

Comment:
I would expect `totalFlatFieldDoubleBytes == 0` for non-value types and so this can be simplified to:
```
	boolean doubleAlignment = (totalDoubleCount > 0) || (totalFlatFieldDoubleBytes > 0);
```